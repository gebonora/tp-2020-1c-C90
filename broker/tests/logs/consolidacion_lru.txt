Partición 1: | 0 - 3 | [X] | Size: 4 b | LRU: 17 | COLA: CAUGHT | ID: 1 |
Partición 2: | 4 - 7 | [X] | Size: 4 b | LRU: 18 | COLA: CAUGHT | ID: 2 |
Partición 3: | 8 - 29 | [X] | Size: 22 b | LRU: 21 | COLA: CATCH | ID: 7 |
Partición 4: | 30 - 46 | [L] | Size: 17 b | LRU: 20 |
Partición 5: | 47 - 62 | [X] | Size: 16 b | LRU: 16 | COLA: CATCH | ID: 5 |
Partición 6: | 63 - 63 | [L] | Size: 1 b | LRU: 15 |
