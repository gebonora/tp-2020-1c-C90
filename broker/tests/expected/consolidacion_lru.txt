Partición 1: | 0 - 3 | [X] | Size: 4 b | LRU: 6 | COLA: CAUGHT | ID: 1 |
Partición 2: | 4 - 7 | [X] | Size: 4 b | LRU: 6 | COLA: CAUGHT | ID: 2 |
Partición 3: | 8 - 29 | [X] | Size: 22 b | LRU: 7 | COLA: CATCH | ID: 6 |
Partición 4: | 30 - 46 | [L] | Size: 17 b |
Partición 5: | 47 - 62 | [X] | Size: 16 b | LRU: 5 | COLA: CATCH | ID: 5 |
Partición 6: | 63 - 63 | [L] | Size: 1 b |
